reference/3.0/Microsoft.PowerShell.Utility/Select-String.md

```diff
@@ -469,8 +468,11 @@ Accept wildcard characters: False
 
 ### -List
 
-Indicates that the cmdlet returns only the first match in each input file. By default,
-`Select-String` returns a **MatchInfo** object for each match it finds.
+Use **List** to retrieve a list of input files whose contents match the regular expression. The
```

Comment:
Change the first paragraph to:

Only the first instance of matching text is returned from each input file. This is the most efficient way to retrieve a list files that have contents matching the regular expression.

Reply:
Thanks. I made the change and pushed the updates for review.